schemas/theia-extension.schema.json

```diff
@@ -0,0 +1,74 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Theia package.json Schema",
+  "type": "object",
+  "$ref": "https://json.schemastore.org/package.json",
```

Comment:
I believe the schema must follow something like this to properly extend the package.json:

```
  "$schema": "http://json-schema.org/draft-07/schema#",
  "title": "Theia package.json Schema",
  "type": "object",
  "allOf": [
    {
      "$ref": "https://json.schemastore.org/package"
    },
    {
      "required": [
        "keywords",
        "theiaExtensions"
      ],
      "properties": {
          ...
      }
    }
  ]
}